### ISSUE TITLE
Block nitter.it instance

### ISSUE THREAD

**Issue by sigaloid** [chr15m/twiiit.com#3]
Whenever twiiit redirects me to nitter.it, it doesn't redirect to the username, just the home page. However, visiting the nitter.it/[username] page directly works as intended. For this reason nitter.it should be blocked from twiiit because there seems to be an issue with their URLs redirecting. 

Test this yourself by loading "https://twiiit.com/ok" a few times and wait for nitter.it to be returned. It does not work as intended and just redirects to the home page.

**Comment by hho** [comment]
I agree. Also, nitter.nixnet.services _seems_ up, but returns a "No items found" page in 99% of my attempts.

But I guess we'd need to remove those from the wiki, if they don't work properly?

**Comment by chr15m** [comment]
The fix here is to check the contents of the page using an HTTP referrer that tells the destination server there has been a redirect from twiiit.com and then inspect the results to make sure they are valid. This would replace the simple check at `/jack`.

**Comment by chr15m** [comment]
PRs welcome!

**Comment by datanomi** [comment]
Hey, owner of the nitter.it instance.
After spending way too much time figuring out what was wrong, I finally figured it out.
The problem was that twiiit.com redirects first to my old domain nitter.mha.fi which then directs to nitter.it
Problem was, nitter.mha.fi didn't return the path, only the domain. 
So I have now fixed the issue, I should also point out to replace the nitter.mha.fi domain with nitter.it.
